st2common/st2common/operators.py

```diff
@@ -186,6 +186,14 @@ def nexists(value, criteria_pattern):
     return value is None
 
 
+def inside(value, criteria_pattern):
+    return value in criteria_pattern
```

Comment:
You're missing the `if criteria_pattern is None` checks that all the other functions using this parameter have. Is this intentional, like it will always be OK ?